Demo/Demo/DemoViewsTableViewController.swift

```diff
@@ -7,16 +7,26 @@ import FinniversKit
 // MARK: - DemoViewsTableViewController
 
 class DemoViewsTableViewController: UITableViewController {
+    lazy var selectorTitleView: SelectorTitleView = {
+        let titleView = SelectorTitleView(withAutoLayout: true)
+        titleView.delegate = self
+        return titleView
+    }()
+
+    private var bottomSheet: BottomSheet?
+
     init() {
         super.init(style: .grouped)
-    }
 
-    required init?(coder aDecoder: NSCoder) { fatalError("") }
+        NotificationCenter.default.addObserver(self, selector: #selector(userInterfaceStyleDidChange(_:)), name: .DidChangeUserInterfaceStyle, object: nil)
+    }
 
-    override var prefersStatusBarHidden: Bool {
-        return true
+    deinit {
+        NotificationCenter.default.removeObserver(self, name: .DidChangeUserInterfaceStyle, object: nil)
```

Comment:
I don't think this is needed since iOS 9

Reply:
You are right! For anyone feeling curious check "Automatic Removal of Finalized Key-Value Observers When Running Garbage-Collected (New since November seed)".

Reply:
https://developer.apple.com/library/archive/releasenotes/Foundation/RN-FoundationOlderNotes/index.html#X10_11Notes